.github/semgrep/rule.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
```

Comment:
What does this rule do? 
Still some strings in `ConfirmationDialog`

Reply:
finds clear text definitions in JSX closures